### DISCUSSION TITLE
Wanted: Publication Strategy

### DISCUSSION THREAD

**Discussion by ThomasLandauer** [Codeception/Codeception#6080] [General]
I just stumbled upon https://github.com/Codeception/Codeception/discussions/6078 and my conclusion is: These are the things that happen when you have too many communication channels and no overview anymore ;-)

Some examples:
* The above discussion (Codeception vs. Behat) is a very interesting question! I asked it myself when I was looking for a testing framework several years ago. **But it should never have been published as a blog post!** Why? => See below.
* On https://twitter.com/davert/status/1290366788300087299 @DavertMik links to his own article on https://medium.com/@docler/testing-microservices-in-php-with-codeception-94e47f22cc24 where he refers people to his gist at https://gist.github.com/DavertMik/5caf209236356ce6c2af036de6d06cdd. WTF!?
* There used to be this "club" website - was it http://www.phptest.club/ (down now)?

My suggestions:

* Everything which is important to *anybody* should go on codeception.com first. This applies to the Behat comparison and the microservices article from above (more precise: The gist maybe a module? And "Testing Microservices" a heading on https://codeception.com/docs/04-FunctionalTests with a link to medium.com).
  Why? Because (a) people will find it, and (b) it will get updated.
* The blog should be merged with https://twitter.com/codeception: Short information for *heavy users*, e.g. "Version x.y released with those 4 cool new features: ..." with links to documentation.

Don't get me wrong: It's great that there's so much information available on so many channels! It's just that people get lost if there's no clear entry point. And this can only be codeception.com

**Comment by DavertMik** [comment]
Thanks @ThomasLandauer 
You are absolutely right, there must be a publication strategy, and sure it went quite differently at different times. For instance, there was no medium.com in 2012.

> inks to his own article on https://medium.com/@docler/testing-microservices-in-php-with-codeception-94e47f22cc24 where he refers people to his gist at https://gist.github.com/DavertMik/5caf209236356ce6c2af036de6d06cdd. WTF!?

That post was sponsored by a company and belongs to the company's blog.

But I agree on most of points. Unfortunately, I can't fix them on my own. 

I'm absolutely ok to give some initiative in this. If you would like to take some part in implementing the content strategy I am absolutely open to sharing all public channels to make it better. 

Even getting up phptest.club is a problem for me  😃

**Reply by ThomasLandauer** [reply]
Spreading the word is certainly good! As a rule of thumb I would suggest:
* Any external article should contain a link to codeception.com. If the topic isn't covered there yet, then include it :-)
* If an external article contains in-depth information that can't be fully integrated on codeception.com, there should be at least a note about the topic and a link to the article.
